Source/WebCore/DerivedSources.make

```diff
@@ -1884,9 +1875,10 @@ MODERN_MEDIA_CONTROLS_STYLE_SHEETS = \
     $(WebCore)/Modules/modern-media-controls/controls/status-label.css \
     $(WebCore)/Modules/modern-media-controls/controls/text-tracks.css \
     $(WebCore)/Modules/modern-media-controls/controls/time-label.css \
+	$(WebCore)/Modules/modern-media-controls/controls/vision-media-controls.css \
+	$(WebCore)/Modules/modern-media-controls/controls/vision-slider.css \
```

Comment:
Are these double-indented on purpose?

Reply:
I think github is lying to you

Reply:
it's not, this looks tab-indented, the other lines are space indented

Reply:
INTERESTING. I guess my github is configured to use 4 space tabs? (and my editor certainly is). I thought Makefiles were always tab-indented, though...